core/src/main/resources/org/apache/spark/ui/static/utils.js

```diff
@@ -46,3 +46,25 @@ function formatBytes(bytes, type) {
     var i = Math.floor(Math.log(bytes) / Math.log(k));
     return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
 }
+
+function padZeroes(num) {
+  return ("0" + num).slice(-2);
```

Comment:
`num.toString().padStart(2, "0")` cause [org.apache.spark.deploy.history.HistoryServerSuite](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/83813/testReport/) test failed.

Reply:
do you know why?

Reply:
`padStart` with poor compatibility:

Feature | Chrome | Edge | Firefox | Internet Explorer | Opera | Safari
-- | -- | -- | -- | -- | -- | --
Basic support | 57 | 15 | 48 | No | 44 | 10


https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/slice

Reply:
what does the test use?

Reply:
[TagNameQuery("a") ](https://github.com/apache/spark/blob/4a78965c22f11fbda7c9ba843ee266048bf6d319/core/src/test/scala/org/apache/spark/deploy/history/HistoryServerSuite.scala#L349) gets result from [WebBrowser](https://github.com/apache/spark/blob/4a78965c22f11fbda7c9ba843ee266048bf6d319/core/src/test/scala/org/apache/spark/deploy/history/HistoryServerSuite.scala#L43), This browser doesn't support this function.